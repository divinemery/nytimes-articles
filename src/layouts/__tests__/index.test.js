import BasicLayout from '..';
import renderer from 'react-test-renderer';

describe('Layout: BasicLayout', () => {
  it('Render correctly', () => {
    const tree = renderer.create(<BasicLayout />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});

