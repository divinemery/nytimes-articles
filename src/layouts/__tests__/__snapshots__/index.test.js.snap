// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Layout: BasicLayout Render correctly 1`] = `
<section
  className="main ant-layout"
>
  <header
    className="ant-layout-header"
  />
  <section
    className="ant-layout ant-layout-has-sider"
  >
    <aside
      className="ant-layout-sider ant-layout-sider-dark"
      style={
        Object {
          "flex": "0 0 200px",
          "maxWidth": "200px",
          "minWidth": "200px",
          "width": "200px",
        }
      }
    >
      <div
        className="ant-layout-sider-children"
      />
    </aside>
    <main
      className="ant-layout-content"
    />
    <aside
      className="ant-layout-sider ant-layout-sider-dark"
      style={
        Object {
          "flex": "0 0 200px",
          "maxWidth": "200px",
          "minWidth": "200px",
          "width": "200px",
        }
      }
    >
      <div
        className="ant-layout-sider-children"
      />
    </aside>
  </section>
</section>
`;
